splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/FromFinalTableIT.java

```diff
@@ -44,6 +48,8 @@
 public class FromFinalTableIT extends SpliceUnitTest {
```

Comment:
i'm not exactly sure why you don't have to specify `@Category(value = {SerialTest.class})` here. but looks like this already runs only in serial. otherwise the assert in line 153 with checkNumTables() would cause sporadics. Maybe you want to add the SerialTest.class as a precaution still?

Reply:
Thanks.  Maybe it's not showing for you, but line 46 has this:
> @Category(value = {SerialTest.class})

Reply:
ah thanks, i missed that it was retracted by github